code/datums/gamemode/role/vampire_role.dm

```diff
@@ -61,11 +60,14 @@
 	for(var/type_VP in roundstart_powers)
 		var/datum/power/vampire/VP = new type_VP
 		VP.Give(src)
+	
+	faction.name = "Lord [antag.current]'s vampiric servants."
```

Comment:
Doesn't work - might change to "Vampire force #[i]"

Reply:
Do it in the faction.